README.md

```diff
@@ -31,6 +31,97 @@ Emmy library is named after a German mathematician [Emmy Noether](https://en.wik
 - [Further documentation](#documentation)
 <!-- tocstop -->
 
+# emmy - anonymous credentials showcase
```

Comment:
This section needs some updating, according to the changes from 9f25aa2b556a61063ed13990da3901b3fb7b8add